go/enclave/core/rollup.go

```diff
@@ -24,7 +24,7 @@ type Rollup struct {
 
 // Hash returns the keccak256 hash of b's header.
 // The hash is computed on the first call and cached thereafter.
-func (r *Rollup) Hash() common.L2RootHash {
+func (r *Rollup) Hash() *common.L2RootHash {
```

Comment:
I changed this to return a pointer, as otherwise in `RollupChain` I often had to do stuff like:

```
hash := rollup.Hash()
return &hash
```